packages/core/package.json

```diff
@@ -36,14 +37,14 @@
   "repository": "https://github.com/wireapp/wire-web-packages/tree/master/packages/core",
   "scripts": {
     "build": "yarn build:node && yarn build:browser",
-    "build:browser": "webpack --progress",
+    "build:browser": "webpack --mode=production --progress",
     "build:node": "tsc",
     "clear": "rimraf dist && rimraf .tmp",
     "coverage": "istanbul cover --report html ./node_modules/jasmine/bin/jasmine.js",
     "dist": "yarn clear && yarn build",
     "start": "rimraf .tmp && yarn dist && node index.js",
     "test": "yarn dist && yarn test:node && yarn test:browser",
-    "test:browser": "webpack && karma start",
+    "test:browser": "webpack --mode=development && karma start",
```

Comment:
Let's also test in production mode if possible

Reply:
👍